src_compressible/boundary.cpp

```diff
@@ -1450,9 +1521,13 @@ void StochFlux(std::array<MultiFab, AMREX_SPACEDIM>& faceflux_in,
                 amrex::ParallelFor(b, AMREX_SPACEDIM, [=] AMREX_GPU_DEVICE (int i, int j, int k, int n) noexcept
                 {
                     if (n == 0) {
+                        // normal velocity and viscous heating (diagonal)
                         flux(i,j,k,1+n) *= sqrtTwo;
+                        flux(i,j,k,nvars+1) *= sqrtTwo;
                     } else {
+                        // transverse velocity and viscous heating (shear)
                         flux(i,j,k,1+n) *= factor;
+                        flux(i,j,k,nvars+2) *= factor;
```

Comment:
@isriva is this double counting the scaling of the viscous heating (nvars+2) term?  Since this will happen for the n==1 and n==2 cases.

Reply:
@ajnonaka oh this is interesting. There are two contributions to flux(i,j,k,nvars+2) -- xy and xz shear. Each needs to be multiplied by sqrtTwo, So, you are right, in effect only a singe sqrtTwo. I am now thinking what is the best way to do this...

Reply:
It won't matter when factor = 0, but certainly will when factor = sqrtTwo

Reply:
you could wrap it in (n==0) so it only happens once

Reply:
that was a strategy used in some code before (that you modified) to handle total rho in a parallelfor over all species

Reply:
or you could take the loop over species out of this loop and put for (int n=0; n<nspecies) in
and separately call out the nvars+1 and nvars+2 cases

Reply:
yes, that is what I did. fixed it now. good catch!